src/Core/Platform/Mailer/Mailer.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using Bit.Core.Models.Mail;
-using Bit.Core.Services;
```

Comment:
Why does mailer go to it's own namespace and not just in `..Services.Mail`, for example?